### PR TITLE
Added several extensions and removed warnings.

### PR DESCRIPTION
**Pull request by NicolasCombe5555**
Fix redundant public warnings,
Add tinted color method to image extension,
Add dayOfTheWeek to date extension,
Add borders extension to uiview,
Add UIImageView extensions,
Add UITableViewCell extensions

### CODE REVIEW

File: MMExtensions/Classes/Extensions/String+Extensions.swift [hunk 1]
```diff
@@ -53,9 +53,9 @@ extension String {
         return String(self[index] as Character)
     }
     
-    subscript (range: CountableClosedRange<Int>) -> String {
-        let start = index(startIndex, offsetBy: range.lowerBound)
-        let end = index(startIndex, offsetBy: range.upperBound)
-        return String(self[Range(start ..< end)])
-    }
+//    subscript (range: CountableClosedRange<Int>) -> String {
```
Comment (MuthurajMuthulingam) [inline MMExtensions/Classes/Extensions/String+Extensions.swift]: @NicolasCombe5555  Please address commented code here!

**Review by MuthurajMuthulingam — CHANGES_REQUESTED** [review]

File: MMExtensions/Classes/Extensions/String+Extensions.swift (hunk 1 above)
Comment (NicolasCombe5555) [reply MMExtensions/Classes/Extensions/String+Extensions.swift]: Hello @MuthurajMuthulingam,
That subscript was showing these errors:
.Referencing initializer 'init(_:)' on 'Range' requires that 'String.Index.Stride' conform to 'SignedInteger'
.Referencing initializer 'init(_:)' on 'Range' requires that 'String.Index' conform to 'Strideable'

Comment (NicolasCombe5555) [reply MMExtensions/Classes/Extensions/String+Extensions.swift]: I fixed it for you

**Review by MuthurajMuthulingam — APPROVED** [review]

**Review by MuthurajMuthulingam — APPROVED** [review]
LGTM

**Comment by MuthurajMuthulingam** [comment]
@NicolasCombe5555  This is finally released after long wait! I took a while due to personal reasons. Thanks for all the contributions. cheers

**Pull request merged**